src/OpenFga.Sdk.Test/Api/OpenFgaApiTests.cs

```diff
@@ -184,12 +185,34 @@ public async Task ApiTokenSentInHeader() {
             );
         }
 
+        /// <summary>
+        /// Test that ApiToken credentials do not cause reserved header exception
+        /// </summary>
+        [Fact]
+        public void ApiTokenDoesNotSetReservedHeaderInDefaultHeaders() {
```

Comment:
It looks like something went a little wrong with a merge with main as these changes are already present from #152